Enigmatry.Entry.Blueprint.Infrastructure.Tests/Configuration/TestConfigurationBuilder.cs

```diff
@@ -45,29 +46,25 @@ public IConfiguration Build()
         };
 
         configurationBuilder.AddInMemoryCollection(dict);
+        _extraConfiguration?.Invoke(configurationBuilder);
         return configurationBuilder.Build();
     }
-
-    public IConfiguration BuildSchedulerConfiguration()
+    
+    public TestConfigurationBuilder WithSchedulerConfiguration()
     {
-        EnsureParametersBeforeBuild();
-        var configurationBuilder = new ConfigurationBuilder();
-
-        var dict = new Dictionary<string, string?>
+        _extraConfiguration = configurationBuilder =>
         {
-            { "UseDeveloperExceptionPage", "true" },
-            { "DbContext:SensitiveDataLoggingEnabled", "true" },
-            { "DbContext:UseAccessToken", "false" },
-            { "DbContext:ConnectionResiliencyMaxRetryCount", "10" },
-            { "DbContext:ConnectionResiliencyMaxRetryDelay", "0.00:00:30" },
-            { $"ConnectionStrings:{_dbContextName}", _connectionString },
-            { "ApplicationInsights:ConnectionString", "" },
-            { "Scheduling:Host:quartz.scheduler.instanceName", "Enigmatry.Entry.Scheduler" },
-            { "Scheduling:Jobs:CleanOldProductsJob:Cronex", "0 0 0 * * ?" }
-        };
+            var dict = new Dictionary<string, string?>
+            {
+                { "ApplicationInsights:ConnectionString", "" },
+                { "ApplicationInsights:TelemetryProcessor:ExcludedTypes", String.Empty },
```

Comment:
This is a new setting. What is it for?

Reply:
I have checked, it is no longer used but was copied from old project. I will remove it.